pkgs/development/compilers/edk2/default.nix

```diff
@@ -12,7 +25,17 @@ else if stdenv.isAarch64 then
 else
   throw "Unsupported architecture";
 
-edk2 = stdenv.mkDerivation {
+buildStdenv = if stdenv.isDarwin then
+  overrideCC clangStdenv [ clang_9 llvmPackages_9.llvm llvmPackages_9.lld ]
+else
+  stdenv;
+
+buildType = if stdenv.isDarwin then
```

Comment:
```suggestion
buildType = if stdenv.cc.isClang then
```

Reply:
Similar problem with macOS/Linux as above, but I guess we could do `if buildStdenv.cc.isClang`?